.pre-commit-config.yaml

```diff
@@ -65,3 +65,11 @@ repos:
         language: golang
         require_serial: true
         pass_filenames: false
+      - id: golangci-lint
```

Comment:
I think it'd also be nice to have a Make target to run linting as well.

Reply:
Good idea, added in latest commit 👍